### PR TITLE
[Feature] Support eval concate dataset and add tool to show dataset

### PR DESCRIPTION
**Pull request by FreyWang**
## Motivation

1. The pr support eval concate dataset by add an option parameter to Dataset.eval() 
2. Add a script to show origin train set and augumented train set which is modified from mmdetection

## Modification

- Add file `tools/browse_dataset.py`
- Add parameter `gt_seg_maps=None` in `custom.py`, Add `evaluate()` to `ConcatDataset` of  `dataset_wrappers.py`

### CODE REVIEW

**Comment by Junjun2016** [comment]
Hi @FreyWang 
Thanks for your contribution!

File: mmseg/datasets/dataset_wrappers.py [hunk 1]
```diff
@@ -14,12 +15,67 @@ class ConcatDataset(_ConcatDataset):
         datasets (list[:obj:`Dataset`]): A list of datasets.
     """
 
-    def __init__(self, datasets):
+    def __init__(self, datasets, separate_eval=True):
```
Comment (Junjun2016) [inline mmseg/datasets/dataset_wrappers.py:18]: Docstring for `separate_eval`.

**Review by Junjun2016 — COMMENTED** [review]
Could you please provide some numerical results for `ConcatDataset` with `seperate_eval`?

**Comment by Junjun2016** [comment]
Hi @FreyWang 
Please also fix the conflict and merge the upstream master.

**Comment by FreyWang** [comment]
> Hi @FreyWang
> Please also fix the conflict and merge the upstream master.

ok, I will do it today

**Comment by Junjun2016** [comment]
> > Hi @FreyWang
> > Please also fix the conflict and merge the upstream master.
> 
> ok, I will do it today

Please also take a look at [this PR](https://github.com/open-mmlab/mmsegmentation/pull/709) #709.

**Pull request closed without merging**